om-2.0.rdf

```diff
@@ -21442,6 +21479,152 @@
     </rdfs:subClassOf>     
   </owl:Class>
 
+  <!-- Volt-Ampere Ontology -->
+    <om:Unit rdf:about="&om;voltampere">
+    <rdfs:label xml:lang="en">volt-ampere</rdfs:label>
+    <rdfs:label xml:lang="nl">volt-ampere</rdfs:label>
+    <rdfs:comment xml:lang="en">Volt-ampere (VA) is a unit of apparent power in a power system. It represents the total power in an AC circuit, including both the real power (measured in watts) and the reactive power (measured in VARs or kilovolt-ampere reactive).</rdfs:comment>
+    <om:longcomment>VA is commonly used to specify the power rating of electrical equipment such as transformers, generators, and uninterruptible power supplies (UPS). The VA rating indicates the maximum amount of power that the equipment can handle, based on the voltage and current ratings.</om:longcomment>
+    <om:symbol>VA</om:symbol>
+    <rdf:type rdf:resource="&om;SingularUnit"/>
+    <om:hasDimension rdf:resource="&om;power-Dimension"/>
+  </om:Unit>
+
+  <!-- Volt-Ampere Multiples and Submultiples Ontology -->
+
+  <om:PrefixedUnit rdf:about="&om;kilovoltampere">
+    <rdfs:label xml:lang="en">kilovolt-ampere</rdfs:label>
+    <rdfs:label xml:lang="nl">kilovolt-ampere</rdfs:label>
+    <rdfs:comment xml:lang="en">The kilovolt-ampere is a unit of apparent power defined as 1.0e3 volt-ampere.</rdfs:comment>
+    <om:symbol>kVA</om:symbol>
+    <om:hasPrefix rdf:resource="&om;kilo"/>
+    <!-- hasSingularUnit --> <om:hasUnit rdf:resource="&om;voltampere"/>
+    <om:hasDimension rdf:resource="&om;power-Dimension"/>
+  </om:PrefixedUnit>
+
+  <om:PrefixedUnit rdf:about="&om;megavoltampere">
+    <rdfs:label xml:lang="en">megavolt-ampere</rdfs:label>
+    <rdfs:label xml:lang="nl">megavolt-ampere</rdfs:label>
+    <rdfs:comment xml:lang="en">The megavolt-ampere is a unit of apparent power defined as 1.0e6 volt-ampere.</rdfs:comment>
+    <om:symbol>mVA</om:symbol>

```

Comment:
"mVA" is milli-volt-ampere. Mega-volt-ampere is "MVA". Please correct.

Reply:
Corrected.